runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsOverlay.java

```diff
@@ -40,28 +47,38 @@
 import net.runelite.client.ui.overlay.components.LineComponent;
 import net.runelite.client.ui.overlay.components.PanelComponent;
 import net.runelite.client.ui.overlay.components.TitleComponent;
+import net.runelite.client.ui.overlay.components.ImageComponent;
+import net.runelite.client.ui.overlay.components.ComponentOrientation;
+import net.runelite.client.ui.overlay.components.ComponentConstants;
+import net.runelite.client.util.ImageUtil;
 
 public class RaidsOverlay extends Overlay
 {
 	private static final int OLM_PLANE = 0;
+	private static final int SMALL_IMAGE_SIZE = 24;
+	private static final int REGULAR_IMAGE_SIZE = 32;
+	private static final int BOLD_IMAGE_SIZE = 27;
```

Comment:
Is it intentional for the bold image size to be smaller than the regular image size?

Reply:
The text it sits next to would be so much larger with bold that I had to make the images smaller, and dynamically changing the panel size was an absolute bear that i stopped trying to fight after a few days and went with this.

Reply:
Is the overlay smaller width with the small font? If not, then why is the size larger at regular size? (If it is, say so and dismiss this comment)

Reply:
The way the base panel is made, is it has a default minimum size which only the bold font really shows.  The small is set because the height of the panel is fully dynamic with the font and so if the images arnt scaled for that they fall off the bottom, and again changing that was difficult for it to do well so i just went with making them smaller for that case too.